static/app/views/alerts/rules/metric/details/anomalyDetectionFeedbackBanner.tsx

```diff
@@ -0,0 +1,106 @@
+import {useDismissable} from 'sentry/components/banner';
+import {Button} from 'sentry/components/button';
+import {feedbackClient} from 'sentry/components/featureFeedback/feedbackModal';
+import {IconClose} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import ConfigStore from 'sentry/stores/configStore';
+import type {Organization} from 'sentry/types/organization';
+import {trackAnalytics} from 'sentry/utils/analytics';
+import type {Incident} from 'sentry/views/alerts/types';
+
+interface AnomalyDetectionFeedbackProps {
+  id: string;
+  organization: Organization;
+  selectedIncident: Incident;
+}
+
+export default function AnomalyDetectionFeedbackBanner({
+  id,
+  organization,
+  selectedIncident,
+}: AnomalyDetectionFeedbackProps) {
+  const [isSubmitted, submit] = useDismissable(id);
+
+  const handleClick = useCallback(
+    (isAnomaly: boolean) => {
+      trackAnalytics('anomaly_detection.submitted_feedback', {
+        choice_selected: isAnomaly,
+        organization,
+        incident_id: id,
+      });
+      feedbackClient.captureEvent({
+        request: {
+          url: window.location.href, // gives the full url (origin + pathname)
+        },
+        tags: {
+          featureName: 'anomaly-detection-alerts-feedback',
+          choice_selected: isAnomaly,
+          incident_id: id,
+          alert_rule_id: selectedIncident.alertRule.id,
+          metric: selectedIncident.alertRule.query,
+          sensitivity: selectedIncident.alertRule.sensitivity,
+          direction: selectedIncident.alertRule.thresholdType,
+          time_window: selectedIncident.alertRule.timeWindow,
+        },
+        user: ConfigStore.get('user'),
+        level: 'info',
+        message: 'Anomaly Detection Alerts Banner Feedback',
+      });
+      submit();
+    },
+    [
+      id,
+      organization,
+      selectedIncident.alertRule.id,
+      selectedIncident.alertRule.query,
+      selectedIncident.alertRule.sensitivity,
+      selectedIncident.alertRule.thresholdType,
+      selectedIncident.alertRule.timeWindow,
+      submit,
+    ]
+  );
+
+  if (isSubmitted) {
+    return null;
+  }
+  return (
+    <StyledAlert
+      type="info"
+      trailingItems={
+        <Fragment>
+          <StyledButton onClick={() => handleClick(true)}>{t('Yes')}</StyledButton>
+          <StyledButton onClick={() => handleClick(false)}>{t('No')}</StyledButton>
+        </Fragment>
+      }
+      showIcon
+    >
+      {t('Was the anomaly correctly identified?')}
+    </StyledAlert>
+  );
+}
+
+const StyledButton = styled(Button)`
+  background-color: transparent;
+  border: 1px solid ${p => p.theme.alert.info.border};
+`;
+
+const StyledAlert = styled(Alert)`
+  margin: 0;
+`;
+
+const CloseButton = styled(Button)`
+  cursor: pointer;
+  z-index: 1;
+`;
+
+CloseButton.defaultProps = {
+  icon: <IconClose />,
+  ['aria-label']: t('Close'),
+  priority: 'link',
+  borderless: true,
+  size: 'xs',
+};
```

Comment:
No longer used?

Reply:
Yes, I checked the designs and there's no close button so I removed it (easier to style without the close button). I'll get rid of this code.